### PR TITLE
Fix installation guide and add mention about it to release documentation

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

File: docs/installer/lokoctl.md [hunk 1]
```diff
@@ -17,19 +17,19 @@ These binaries can be manually downloaded and installed.
    keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).
 
 ```console
-gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig
+gpg --verify lokoctl_0.5.0_linux_amd64.tar.gz.sig
```
Comment (ipochi) [inline docs/installer/lokoctl.md:20]: Why have a hard coded number ?

```suggestion
gpg --verify lokoctl_0.X.Y_linux_amd64.tar.gz.sig
```

and point to the releases page. This way master branch will have the `0.x.y` reference and each release tag branch will have the correct version.

**Review by ipochi — CHANGES_REQUESTED** [review]

File: docs/installer/lokoctl.md (hunk 1 above)
Comment (invidian) [reply docs/installer/lokoctl.md:20]: Why not X.Y.Z then? I think having a hardcoded number is better, as it's something you can copy-paste to the terminal and run.

Comment (ipochi) [reply docs/installer/lokoctl.md:20]: Yes its true its better but not for main branch.

I am fine with keeping the latest release number in the main branch if we add this task to the release checklist of updating everytime.

**Review by ipochi — APPROVED** [review]
Thanks for the PR @invidian 

LGTM :tada:

**Pull request merged**